Dockerfile

```diff
@@ -56,10 +60,10 @@ RUN echo "export VISIBLE=now" >> /etc/profile
 
 #configs bash start
 COPY configs/autostart.sh /root/autostart.sh
-RUN  chmod +x /root/autostart.sh
+COPY configs/chown.sh /root/chown.sh
+RUN chmod +x /root/autostart.sh
+RUN chmod +x /root/chown.sh
```

Comment:
Do not need to run shell script to change owner.
Just write this to dockerfile
RUN chown -R docker:www-data /var/www